Modules/III-k6-Intermediate/01-How-to-debug-k6-load-testing-scripts.md

```diff
@@ -146,10 +145,7 @@ INFO[0000] username: guest  / password: 12345          source=console
 
 That way, you know exactly which combination to try. Perhaps the username or the password is incorrect, or perhaps both. Either way, adding logs to your script help you understand the value of key variables your script uses.
 
-```ad-warning
-title: Disable logging during load tests
-`console.log()` can be very resource-intensive, and too much logging can affect your test results. Comment out logging as much as possible to avoid high resource utilization during test execution.
-```
+> :warning: **Disable logging during load tests**. `console.log()` can be very resource-intensive, and too much logging can affect your test results. Comment out logging as much as possible to avoid high resource utilization during test execution.
```

Comment:
```suggestion
> :warning: **Disable logging during load tests**. `console.log()` can be very resource-intensive, and too much logging can affect your test results. Comment out logging as much as possible to avoid high resource utilization as the test runs.
```

Reply:
Just as a general principle, I always try minimize the number of nouns, and choose the shortest words possible (these guidelines are related :smiley: ).